cli/README.md

```diff
@@ -1,55 +1,74 @@
 
-In order to configure authentication for the _global mode_, you have to authenticate first using `xata auth login`. This will guide you through the process to generate a new global API key that will be stored in `~/.config/xata/credentials`.
+- **Create a new API Key** will open your browser and, when you're logged in to Xata, allow you to create a new [API key](https://docs.xata.io/concepts/api-keys) for use with the CLI.
+- **Use an existing API Key** will prompt for an existing key you have, which you can paste into your terminal. We recommend isolating keys per use-case, so creating a new one might be a good idea here.
 
-## Configuring a project
+## Initializing a Project
 
-In ordert to configure the CLI in a project you'll run `xata init --db=https://{workspace}.xata.sh/db/{database}`. The flag `--db` is not mandatory but recommended, because this way you won't be asked for the workspace/database and thus, a global API key is not required.
+To initialize a new database, run `xata init` in your project's directory.
 
-This command will ask you a few questions and will run a few operations:
+- if you don't have a [workspace](https://docs.xata.io/concepts/workspaces), you will be prompted to create one.
+- if you have workspaces, you'll be asked to choose one, or to create a new one.
 
-- It will create a project configuration file (`.xatarc`).
-- It will create or update your `.env` file to store a database-scoped API key. (At the time of writing these docs it'll actually store a personal API key, but this will change soon).
-- It will ask you if you want to install the SDK and/or use the TypeScript/JavaScript code generator.
+A workspace is a logical grouping of databases, usually analogous to an organization or team, so this is the first step. Once you've chosen a workspace, you will be given the option to either create a new one, or use an existing one for your project.
 
-## Code generation
+After choosing a workspace and a database, you're ready to go: the CLI will walk you through next steps, ultimately leading you to code generation.
 
-You can use the Xata SDK (`@xata.io/client`) directly or with code generation. The benefit of using code generation is that you'll have a Xata client with specific types and classes for your database schema. This is great for TypeScript because it ensures type safety, but also for JavaScript because you can benefit from its type definitions if you use development tools such as IDEs or text editors that support them.
+## Code Generation
 
-Code generation will be executed automatically in some cases by the CLI (such as when creating a new Xata branch). You can always invoke it directly with `xata codegen`.
+We strongly recommend using the CLI to generate your Xata fetcher (sometimes called a _client_) for you. This will help make your queries to Xata predictable and safe, while also providing autocomplete and safety.
```

Comment:
We are using the naming `XataClient` in other places. I prefer that than `Xata fetcher`. Specially because `fetcher` sounds like you can only fetch data, and not mutate it. Besides, we are offering a fully fledged ORM, `fetcher` feels like something small, like a utility function.

Reply:
Thanks, I hear that. The intent is more to speak to developers who are familiar with `fetch` (a lot) speaking their language and making a mental association with how it works: `fetch` can be used to both read and mutate data, just like the client. 

Meanwhile, my fear of using just the term “client” with no mental association otherwise is that we use “client” in multiple ways:

- a client application to a remote server, like the Xata Web UI
- the Spotify app on a smart TV is a Spotify client
- what we call a client here 

Using the term `fetch`, then clarifying straight after that this is sometimes called a client, is our way of clarifying our intent with the client word. 

Does that make sense? Or should we still change it like this do you think?